lib/src/types.rs

```diff
@@ -40,7 +40,7 @@ impl Default for Class {
 pub struct FieldDecl {
     pub field_type: Type,
     pub name: String,
-    pub val: Option<String>, // @Decide: Should probably Option<Expr> instead
+    pub val: Option<Expr>,
```

Comment:
According to our teacher, perhaps we should consider removing the value field completely

![image](https://github.com/Java-rs/RustyJ/assets/91947480/1c683bdc-47d8-4b64-a98d-3bd52b0fbddb)

Reply:
We need the value field if we want to support the following assignments:
```java
class A {
    int x = 9;
}
````
Without the value field, we can't represent the `= 9`. I know our teacher didn't have that in his slides, but the things in his slides were partially incomplete and partially covering things we aren't actually required to do, so I would prefer to diverge from his slides in this case.

Reply:
Sure, I had overlooked that. Then of course we need that.